stratum/glue/net_util/ipaddress.cc

```diff
@@ -736,7 +736,6 @@ std::string SocketAddress::ToString() const {
 
 std::string SocketAddress::ToPackedString() const {
   LOG(FATAL) << __FUNCTION__ << " not yet implemented in depot3";
```

Comment:
some idea to implement this function

according to the doc:
```
Returns the address as a sequence of bytes in network-byte-order, IPAddress first, then port.
```

```c++
std::string SocketAddress::ToPackedString() const {
  std::string ip_addr_str = host_.ToPackedString();
  char tmp[ip_addr_str.size() + 2]; // IP address len + L4 port len
  strncpy(tmp, ip_addr_str.c_str(), ip_addr_str.size());
  tmp[ip_addr_str.size()] = (char)(port_ >> 8);
  tmp[ip_addr_str.size()+1] = (char)(port_ & 0xff);
  return std::string(tmp, ip_addr_str.size()+2);
}
```

Reply:
Thanks for picking this up and trying to fix the root cause!

If you want, we should open another PR for this.

Reply:
We even have the tests for it https://github.com/stratum/stratum/blob/9f5bd2b285badbef11e81eca6c31d4a3c4342843/stratum/glue/net_util/ipaddress_test.cc#L1186-L1276